src/app/content/practiceQuestions/types.ts

```diff
@@ -31,8 +32,17 @@ export interface PracticeQuestion {
 
 export type PracticeQuestions = PracticeQuestion[];
 
+export interface QuestionsAndAnswers {
```

Comment:
would this be better named QuestionAnswers? since it doesnt actually include the question